src/install.c

```diff
@@ -242,7 +242,7 @@ static gchar** get_all_manifest_slot_classes(const RaucManifest *manifest)
  * Note: This function may be extended to be more sophisticated or follow a
  * certain policy for selecting an appropriate slot!
  *
- * @param rootclass name of root slot class
+ * @param rootclass name of root slot class (must be an intern string)
```

Comment:
As we can't easily check if a string is interned, this seems a bit too fragile for the potential performance gain.

Reply:
Yes, I also already noted that this would require being extra careful.
Thus if at all, It's probably only useful for memory allocations. I've removed the corresponding patch from the set.